openfl/utilities/workspace.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Workspace utils module."""
+
+import logging
+import os
+import shutil
+import sys
+import time
+from pathlib import Path
+from subprocess import check_call
+from sys import executable
+
+logger = logging.getLogger(__name__)
+
+
+class ExperimentWorkspace:
+    """Experiment workspace context manager."""
+
+    def __init__(
+            self,
+            experiment_name: str,
+            data_file_path: Path,
+            is_install_requirements: bool = False
```

Comment:
is_install_requirements -> install_requirements
probably this will look better

Reply:
It's boolean flag. It will be strange:
```python
if self.install_requirements:
            self._install_requirements()
```
Probably `is_install_requirements_required` would be better, but too long.

Reply:
I also think that `requirements` will be better. You still can rename it inside `__init__`.